mleap-xgboost-runtime/src/test/scala/ml/combust/mleap/xgboost/runtime/XGBoostPredictorClassificationModelParitySpec.scala

```diff
@@ -87,13 +95,13 @@ class XGBoostPredictorClassificationModelParitySpec extends FunSpec
   }
 
   it("[Multinomial] An XGBoost4j Booster has the same results as a deserialized Predictor"){
-    equalityTestRowByRowMultinomialProbability(ClassifierUtils.multinomialBooster, ClassifierUtils.deserializedMultinomialXGBoostPredictor, leapFrameMultinomial)
+    equalityTestRowByRowMultinomialProbability(ClassifierUtils.multinomialBooster, ClassifierUtils.deserializedMultinomialXGBoostPredictor, leapFrameMultinomial, precision = 1e-5)
   }
 
   it("[Multinomial] XGBoostPredictorMultinomialClassificationModel results are the same pre and post serialization") {
     val xgboost4jResult = ClassifierUtils.multinomialMleapTransformer.transform(leapFrameMultinomial).get
     val predictorResult = ClassifierUtils.deserializedMultinomialXGBoostPredictor.transform(leapFrameMultinomial).get
-    probabilityColumnEqualityTest(xgboost4jResult, predictorResult)
+    probabilityColumnEqualityTest(xgboost4jResult, predictorResult, precision=1e-5)
```

Comment:
The default precision limit (1e-7) will break the test under latest xgboost version.

Reply:
This makes me a little nervous.  Perhaps we also need to upgrade xgboost predictor library?

Reply:
xgboost predictor library is already the newest version.
 the precision=1e-5 i think it is fine. New version xgboost algorithm might change slightly so it affect the precision but very small difference.

Reply:
I guess the question is more whether the xgboost predictor library itself needs some PR to upgrade it for xgboost 1.5.x support?

Agreed that some change to xgboost might have changed it's decisions a little bit, but the model predictions still should be reproducible and change in the same way.  These tests are just ensuring that the pre- and post-serialized models give the same results.

I tend to get nervous about these xgboost precision details because I have previously encountered issues where little tiny differences in the xgboost setup/usage can impact the predictions in major ways (e.g., whether you use 32 or 64 bit floats).  However, since our testing model/data are simple they might only demonstrate small changes to output, while a more real-world model/data would demonstrate predictions being completely broken.

Reply:
@jsleight 
I test again. It turns out seems previously I test against wrong predictor version.
Current PR when I increase prediction tolerance to 1e-7 it still pass test.

Reply:
I revert the change here.

Reply:
awesome!  Thanks for reconfirming ❤️